snapshotTestContent/DeletedTables_0.43/current_snapshots/snapshot_461_0.json

```diff
@@ -807,8 +897,7 @@
                       "encoding": "utf-8"
                     }
                   }
-                ],
-                "unreferenced": true
```

Comment:
@agarwal-navin i see these unreferenced flags going away. i think it's because we not get the sparse matrix which has the references that were previously missing. so should increase GC coverage :)

Reply:
That's good. The content changes because for SharedMatrix, now we get the actual factory and just don't compare it, right.